src/infrastructure/repositories/figma-team-repository.integration.test.ts

```diff
@@ -23,21 +23,42 @@ describe('FigmaTeamRepository', () => {
 					),
 				]);
 			const [targetFigmaTeam1, targetFigmaTeam2] = await Promise.all([
-				figmaTeamRepository.upsert(
-					generateFigmaTeamCreateParams({
-						connectInstallationId: targetConnectInstallation.id,
-					}),
-				),
-				figmaTeamRepository.upsert(
-					generateFigmaTeamCreateParams({
-						connectInstallationId: targetConnectInstallation.id,
-					}),
-				),
-				figmaTeamRepository.upsert(
-					generateFigmaTeamCreateParams({
-						connectInstallationId: anotherConnectInstallation.id,
-					}),
-				),
+				figmaTeamRepository
+					.upsert(
+						generateFigmaTeamCreateParams({
+							connectInstallationId: targetConnectInstallation.id,
+						}),
+					)
+					.then((team) =>
+						figmaTeamRepository.getByTeamIdAndConnectInstallationId(
+							team.teamId,
+							targetConnectInstallation.id,
+						),
+					),
+				figmaTeamRepository
+					.upsert(
+						generateFigmaTeamCreateParams({
+							connectInstallationId: targetConnectInstallation.id,
+						}),
+					)
+					.then((team) =>
+						figmaTeamRepository.getByTeamIdAndConnectInstallationId(
+							team.teamId,
+							targetConnectInstallation.id,
+						),
+					),
+				figmaTeamRepository
```

Comment:
Do we need the 3rd upsert? Looks like we're only using the first two

Reply:
lol i just used a vim macro to update all the tests that were failing - we don't technically need this but the symmetry is nice i guess